packages/expo-updates/ios/EXUpdates/AppLoader/EmbeddedAppLoader.swift

```diff
@@ -129,35 +128,10 @@ public final class EmbeddedAppLoader: AppLoader {
   }
 
   override public func downloadAsset(_ asset: UpdateAsset) {
-    let destinationUrl = directory.appendingPathComponent(asset.filename)
+    let destinationPath = UpdatesUtils.path(forBundledAsset: asset) ?? ""
```

Comment:
prefer to handle the nil case explicitly somehow. something like

```suggestion
    guard let destinationPath = UpdatesUtils.path(forBundledAsset: asset) else {
      // do something when this is nil. it's less clear what the behavior for `fileExists(atPath: "")` is 
    }
```

Reply:
I'll have a look at the best way to handle this.